vendor/bat-native-ledger/src/bat/ledger/internal/publisher/channel_response.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+syntax = "proto3";
+
+option optimize_for = LITE_RUNTIME;
+
+package publishers_pb;
+
+message SocialLinks {
```

Comment:
Are GitHub and Reddit links handled as URLs?

Reply:
Yes, they are either URLs or empty strings.